ipam/space/space.go

```diff
@@ -96,35 +112,31 @@ func (s *Space) Free(addr address.Address) error {
 	return nil
 }
 
-func (s *Space) biggestFreeRange() (int, address.Offset) {
-	pos := -1
-	biggest := address.Offset(0)
-
-	for i := 0; i < len(s.free); i += 2 {
-		size := address.Subtract(s.free[i+1], s.free[i])
-		if size >= biggest {
-			pos = i
-			biggest = size
+func (s *Space) biggestFreeRange(r address.Range) (biggest address.Range, biggestSize address.Offset) {
```

Comment:
There's really no need to return `biggestSize` here, since it's easily obtained at the call site with `biggest.Size()`